### DISCUSSION TITLE
can i get writing-mode attritube from svg use fabricjs?

### DISCUSSION THREAD

**Discussion by sanheng** [fabricjs/fabric.js#7908] [Q&A]
<img width="531" alt="image" src="https://user-images.githubusercontent.com/13793256/165317543-bc882c4e-f858-4253-b9f3-7f50a39f7df2.png">

use fabric.SHARED_ATTRIBUTES can not get writing-mode.

`fabric.SHARED_ATTRIBUTES = fabric.SHARED_ATTRIBUTES.concat('writing-mode')`

<img width="575" alt="image" src="https://user-images.githubusercontent.com/13793256/165317801-3c705a36-f977-457f-b743-addf5f2b7d45.png">

**Comment by asturur** [comment]
We still need to start the work for vertical text, to be honest that would be a priority for me.
We have in the pipeline, RTL text and typescript first.

**Reply by sanheng** [reply]
At present, I want to read this attribute and draw vertical text by myself. Is there any way to read this attribute? I use it to judge whether it's vertical text or horizontal text. If it's horizontal text, I won't deal with it, and I'll deal with vertical text in a special way

**Reply by sanheng** [reply]
@asturur 
Hello, which method can I use to read raw transform, writing-mode and some attributes in SVG? I want to do some calculations.

<img width="546" alt="image" src="https://user-images.githubusercontent.com/13793256/165715635-dc77d8f8-495b-401a-bbdc-8988cd46ce4b.png">

**Reply by asturur** [reply]
The transform gets absorbed by normal object properites, while the writing-mode needs to be added to the parser itself.
As of today isn't taken in consideration. You can see where the other attributes are parsed ( file svg_parsers ) and fork your own svg parser from there.
